scripts/userdata.sh

```diff
@@ -1,17 +1,86 @@
 #!/bin/bash
 
-DEBUG=$${DEBUG:=off}
+# Install AWS SSM agent for different OS versions
+os_type=$(uname -m)
+os_release=$(grep -oP '(?<=^ID=).+' /etc/os-release | tr -d '"')
 
-if [ $${DEBUG} == 'on' ]; then
-    set -x
+if [ "$${os_type}" == "x86_64" ]; then
```

Comment:
Are you repeating the agent instalation here?

Reply:
Removed one script and the installation of unwanted packages